model/trace/database.yaml

```
@@ -24,15 +22,45 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If using a port other than the default port for this DBMS and if `server.address` is set.
+      - ref: db.instance.id
+        requirement_level:
+          recommended: If different from the `server.address`
       - ref: network.peer.address
+        brief: Peer address of the database node operation was performed on.
+        requirement_level:
+          recommended: If applicable for this database system.
+        note: >
+          Semantic conventions for individual database systems SHOULD document whether `network.peer.*` attributes are applicable.
+
+          Network peer address and port are important when the application interacts with individual database nodes directly,
+          for example, when database is self-hosted.
+
+          Instrumentations for databases that don't provide self-hosted mode SHOULD NOT set this attribute.
```

"self-hosted" doesn't feel quite right to me, as it seems like peer address could be useful(?) for hosted databases too

maybe saying less here is ok?

```suggestion
          Network peer address and port are useful when the application interacts with individual database nodes directly.
```

---

makes sense, thank you!